samples/encryption.js

```diff
@@ -122,13 +122,34 @@ function downloadEncryptedFile(bucketName, srcFilename, destFilename, key) {
   // [END storage_download_encrypted_file]
 }
 
-// [START storage_rotate_encryption_key]
-function rotateEncryptionKey() {
-  throw new Error(
-    'This is currently not available using the Cloud Client Library.'
-  );
+function rotateEncryptionKey(bucketName, fileName, oldKey, newKey) {
+  // [START storage_rotate_encryption_key]
+  // Imports the Google Cloud client library
+  const Storage = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const fileName = 'Nome of a file in the bucket, e.g. my-file';
+
+  storage
+    .bucket(bucketName)
+    .file(fileName, {
+      encryptionKey: Buffer.from(oldKey, 'base64'),
```

Comment:
As a user I would think the Buffer.from(oldKey, 'base64') would be hidden and I would only consider passing a base64 string without creating a Buffer object. 

Could you provide more clarity here?

Reply:
Our API was written to accept either a Buffer or 32 characters that we make the key from. It does not accept a base64 string, which is why the sample code has to do the `Buffer.from` conversion.